frontend/src/scenes/web-analytics/WebAnalyticsFilters.tsx

```diff
@@ -45,14 +45,25 @@ export const WebAnalyticsFilters = (): JSX.Element => {
                                 <LemonSelect
                                     className="grow md:grow-0"
                                     size="small"
-                                    value={domainFilter || 'all'}
+                                    value={domainFilter ?? (hasHostFilter ? 'host' : 'all')}
```

Comment:
Maybe we should trust the `hasHostFilter` more than the `domainFilter` here?

```suggestion
                                    value={hasHostFilter ? 'host' : domainFilter ?? 'all'}
```